includes/Core/Modules/Modules.php

```diff
@@ -256,7 +257,9 @@ function( Module $module ) {
 					}
 				);
 
-				$data['activeModules'] = array_keys( $non_internal_active_modules );
+				$data['activeModules']          = array_keys( $non_internal_active_modules );
+				$data['psiManuallyActivated']   = $this->manually_enabled( 'pagespeed-insights' ) && $this->options->get( Migration_1_46_0::OPTION_KEY_PSI_UPDATED );
```

Comment:
@ivankruchkoff, this line is not needed anymore, right?

Reply:
https://github.com/google/site-kit-wp/pull/4305/commits/04da3b7f7f3fdb28dea93771cf83620555d4c817